packages/sdk/.drone.yml

```diff
@@ -281,41 +337,7 @@ steps:
 
   ### START LOCAL STACK END ###
 
-  - name: install
-    image: node:18.19
-    pull: always
-    commands:
-      - node -v
-      - npm -v
-      - cd packages/sdk
-      - npm ci
-      - npm run codegen
-
-  - name: format
-    image: node:18.19
-    commands:
-      - cd packages/sdk
-      - npm run check-format
-    depends_on:
-      - install
-
-  - name: lint
-    image: node:18.19
-    commands:
-      - cd packages/sdk
-      - npm run lint
-    depends_on:
-      - install
-
-  - name: check-types
-    image: node:18.19
-    commands:
-      - cd packages/sdk
-      - npm run check-types
-    depends_on:
-      - install
-
-  - name: test
+  - name: test:e2e
     image: node:18.19
     commands:
       - cd packages/sdk
```

Comment:
here we run again unit tests. May be we need also a script to make `npm run test:e2e`?

Reply:
Good point, I've added all possible unit + e2e test scripts to `package.json` and updated this drone config file accordingly.